zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -887,7 +887,7 @@ paths:
         To change the time, the clock must be pinned again with a new timestamp.
 
         :::note
-        This endpoint is an [alpha feature](/reference/alpha-features.md) and may be subject to change
+        This endpoint is an [alpha feature](/components/early-access/alpha/alpha-features.md) and may be subject to change
```

Comment:
💬 Having a docs link in our OpenAPI spec feels....weird. No tool outside of the docs knows what these paths are.

What do we think about removing these links in the source spec, and injecting them as part of the docs generation process (by looking for the text "This endpoint is an alpha feature")?

Reply:
Yep, I totally agree, I had the same issue with this. But it's a general issue with writing OpenAPI descriptions that also look as expected in our docs, e.g. adding notes and warnings for endpoints.

For alpha features, we can of course do something like that, yeah. As an endpoint developer, if you want to have it generated for your endpoint, you can add an `(alpha)` to the end of the path summary.

Reply:
It's not gonna be possible to add more details to the alpha note then for an endpoint developer but we can manage, I think.

For example, the document endpoints provide more details on why they are alpha features still (they use an in-memory store currently, thus they're not production-ready). But this can be a "regular" part of the endpoint description. We can still generate the alpha note at the beginning of the endpoint description automatically.

For the query endpoints, this will be a bit harder because they actually want to link back to the C8 REST API overview docs where the general explanation sits. We don't have to worry too much here, though, this will be removed with 8.7 anyway soon with https://github.com/camunda/camunda/issues/20579 🙂

Reply:
I added a proposal how this could look like with https://github.com/camunda/camunda/pull/24190/commits/0849e8745da86b2d9a0860b5fab641acfa01e2ae.

We can move to generate that alpha note at some point and remove them from the spec itself when we're ready.